### ISSUE TITLE
Add the administration application

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#4]
### Who

- [ ] Anonymous visitor
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [x] Administrator

### Needs what

To run the administration application.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
